main/main.cpp

```diff
@@ -344,6 +344,7 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("  --resolution <W>x<H>                         Request window resolution.\n");
 	OS::get_singleton()->print("  --position <X>,<Y>                           Request window position.\n");
 	OS::get_singleton()->print("  --single-window                              Use a single window (no separate subwindows).\n");
+	OS::get_singleton()->print("  -xr <mode>, --xr-mode <mode>                 Select XR mode (default/off/on).\n");
```

Comment:
`-` shortcuts should be a single letter only. But I don't think any is needed for this, it's not a very commonly used option and it's already short.

Reply:
Yeah `-x` seemed weird.. I'll remove it all together.

Reply:
Done